velox/docs/functions/presto/tdigest.rst

```diff
@@ -0,0 +1,107 @@
+==================
+T-Digest Functions
+==================
+
+Velox implements algorithms for estimating rank-based metrics using `T-digest
```

Comment:
Is this a copy-paste from Presto? This sounds outdated and misleading.

Velox chose to use more modern algorithm (KLL digest) for approx_percentiles. I believe these functions exits purely to support pre-existing workloads that happen to have data stored using T- and Q-digest.

Please, make sure to describe this clearly.

CC: @amitkdutta